package.json

```diff
@@ -53,7 +53,7 @@
     "@types/picomatch": "^3.0.1",
     "@vercel/git-hooks": "^1.0.0",
     "@vercel/style-guide": "^5.2.0",
-    "analytics-node": "^3.4.0-beta.1",
+    "analytics-node": "^3.5",
```

Comment:
Latest is 6.2.0, lets use that instead

Reply:
Will have a look at it - but switch from 3 to 6 might have been breaking stuff

Reply:
Done